archivist_samples/signed_records/README.md

```diff
@@ -0,0 +1,25 @@
+# Signed Records sample
+
+A key aspect of Zero Trust architectures is the removal of inherent trust in any individual part of your network. The best way to assure trustworthy operations is to combine several best-of-breed security technologies and then adopt a trust-but-verify approach to all of them.
+
+One such strong technology is to use device certificates or cryptographic keys on your IoT devices and use them to authenticate messages, proving that the message came from a particular piece of hardware. This of course is not foolproof: a compromised device with a strong key can send some very secure bad messages! But it is a strong defence against simple network-borne attacks like spoofing and man-in-the-middle forgeries.
+
+RKVST already supports high integrity authentication from secure devices trough its flexible command authorization scheme (leading to the 'who' in 'when who did what') but this can be added to by also signing the message contents (the 'what') with a device key. This signature is then verifiable independent of RKVST and provides additional proof of authenticity when it's needed.
```

Comment:
trough --> through referencing below. . . 
secure devices trough its flexible command

Reply:
Addressed locally. Will push when all comments are collected.

Reply:
"who did what when" is the order of things now.

Reply:
Order changed